src/components/Table/Table.module.css

```diff
@@ -0,0 +1,25 @@
+.table {
```

Comment:
Should we avoid using table here since our component name is `<Table />` and our css is scoped?